cmd/eksctl/create.go

```diff
@@ -91,6 +91,7 @@ func createClusterCmd() *cobra.Command {
 	fs.DurationVar(&cfg.WaitTimeout, "timeout", api.DefaultWaitTimeout, "max wait time in any polling operations")
 
 	fs.BoolVar(&cfg.Addons.WithIAM.PolicyAmazonEC2ContainerRegistryPowerUser, "full-ecr-access", false, "enable full access to ECR")
+	fs.BoolVar(&cfg.Addons.Storage, "storage-class", false, "Add a default StorageClass of type gp2 provisioned by EBS")
```

Comment:
From the discussion in #175, it sounds like the flag should be a string which defaults to gp2.  So, something similar to:

`fs.StringVar(&cfg.Addons.Storage, "storage-class", "gp2, "Adds a StorageClass. Defaults to a type gp2 provisioned by EBS")`

So running `eksctl create cluster` would create a default storage class of gp2 but this can be overridden.

@christopherhein @karinnainiguez  - what do you think?

Reply:
Sorry this conversation should have been external. We had a short discussion about the implementation and had recommended following the path that `kops` uses rather than implement the full gamut of storage classes just set a default so out of the box PVs/PVCs work and let users that understand and want to change it do so after the fact with their own settings. `kops` just auto deploys the `gp2` class and calls it a day.

Do think it would be necessary to have all classes configurable at cluster creation?

Reply:
@christopherhein - the makes sense to me and i guess if we where to use another volume type (such as io1) then we'd may have to  add additional parameters to cover things like iopspergb.  Flipping to `true` as default would be good.

Reply:
Awesome!  Thank you @christopherhein and @richardcase for all your help with this! 

I just submitted the change to make the flag true as default.  Let me know if you need me to make any other changes.